Client/Ecosia/UI/NTP/Impact/NTPImpactCellViewModel.swift

```diff
@@ -148,6 +148,10 @@ extension NTPImpactCellViewModel: HomepageViewModelProtocol {
     var isEnabled: Bool {
         User.shared.showClimateImpact
     }
+    
+    func refreshData(for traitCollection: UITraitCollection, isPortrait: Bool, device: UIUserInterfaceIdiom) {
```

Comment:
Did you check if this is in fact called when the NTP appears and not only when it is refresh (which would be for example when the settings change)?

I believe it is, just raising since I'm not sure. Might be safer to do it on `HomepageViewController.viewDidAppear` somehow, though I agree that brings additional complexity.

Reply:
From the tests I made making use of the `refreshData()` accomplishes our scope. Moving it in any other view life cycle observer function won't make it work as the view of the `HomepageViewController` is initialized only once and the view life cycle events won't be called as the view is shown via transitions, not presented.